delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/routes/ResourcesRoutes.scala

```diff
@@ -92,11 +95,11 @@ final class ResourcesRoutes(
                   (post & pathEndOrSingleSlash & noParameter("rev")) {
                     operationName(s"$prefixSegment/resources/{org}/{project}/{schema}") {
                       authorizeFor(ref, Write).apply {
-                        entity(as[Json]) { source =>
+                        entity(as[NexusSource]) { source =>
                           emit(
                             Created,
                             resources
-                              .create(ref, schema, source)
+                              .create(ref, schema, source.value)
```

Comment:
The idea was also to change the create and update method to take a `NexusSource` instead of a json but it can be done in another PR.